pennylane/transforms/dynamic_one_shot.py

```diff
@@ -36,6 +36,8 @@
 
 from .core import transform
 
+fillin_value = np.iinfo(np.int32).min
```

Comment:
```suggestion
fill_in_value = np.iinfo(np.int32).min
```
If I understood it right.